admission-webhook/integration_tests/templates/simple-with-preset-gmsa-container-level-contents.yml

```diff
@@ -0,0 +1,26 @@
+## a simple deployment with a container-level cred spec GMSA's *contents* already set (and not its name)
+
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  labels:
+    app: {{ .TestName }}
+  name: {{ .TestName }}
+  namespace: {{ .Namespace }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: {{ .TestName }}
+  template:
+    metadata:
+      labels:
+        app: {{ .TestName }}
+    spec:
+      serviceAccountName: {{ .ServiceAccountName }}
+      containers:
+      - image: k8s.gcr.io/pause
```

Comment:
`image` points to `pause` but `name` points to nginx. Is this intentional here and elsewhere?